### PR TITLE
Replace np.ceil() with a faster operation

### PR DESCRIPTION
**Pull request by fuzzythecat**
Replacing np.ceil() with a faster operation, as suggested in #8488

### CODE REVIEW

File: examples/mnist_dataset_api.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def cnn_layers(inputs):
 
 batch_size = 128
 buffer_size = 10000
-steps_per_epoch = int(np.ceil(60000 / float(batch_size)))  # = 469
+steps_per_epoch = (60000 + batch_size -1) // batch_size # = 469
```
Comment (fchollet) [inline examples/mnist_dataset_api.py]: This operation is done once, thus shaving nanoseconds on it is not useful. We should optimize for readability. `np.ceil` is more readable in my opinion.

File: examples/mnist_tfrecord.py [hunk 2]
```diff
@@ -207,7 +207,7 @@ def cnn_layers(x_train_input):
 threads = tf.train.start_queue_runners(sess, coord)
 
 train_model.fit(epochs=epochs,
-                steps_per_epoch=int(np.ceil(data.train.num_examples / float(batch_size))),
+                steps_per_epoch=(data.train.num_examples + batch_size - 1) // batch_size,
```
Comment (fchollet) [inline examples/mnist_tfrecord.py]: Same comment.

File: keras/preprocessing/sequence.py [hunk 3]
```diff
@@ -333,9 +333,8 @@ def __init__(self, data, targets, length,
                              % (self.start_index, self.end_index))
 
     def __len__(self):
-        return int(np.ceil(
-            (self.end_index - self.start_index + 1) /
-            (self.batch_size * self.stride)))
+        return (self.end_index - self.start_index + self.batch_size * self.stride) //
+            (self.batch_size * self.stride)
```
Comment (fchollet) [inline keras/preprocessing/sequence.py]: Ok for this one, easier to justify.

**Comment by fuzzythecat** [comment]
@fchollet Thanks for the comments. Changed the commit accordingly.

**Comment by fchollet** [comment]
Will merge after tests pass.

**Comment by fuzzythecat** [comment]
@fchollet Fixed it.

**Pull request merged**